configapp/static/js/utils.js

```diff
@@ -0,0 +1,41 @@
+function util_draw_polygons(canvas, ctx, image, polygons, occupies)
+{
+    try {
+        if (image !== null) {
+            ctx.clearRect(0, 0, canvas.width, canvas.height);
+            ctx.drawImage(image, 0, 0);
+            
+            for (var i = 0; i < polygons.length; i++) {
+                var polygon = polygons[i]['points'];
+
+                ctx.save();
+                ctx.beginPath();
+                ctx.strokeStyle = '#0AFC04';
```

Comment:
@Adman Should this value be configurable (along with the `'red'` and `"rgba(255, 0, 0, 0.2)"` below)?